.nvmrc

```diff
@@ -0,0 +1 @@
+v10.19.0
```

Comment:
please add the end white line for each file.

Reply:
Donde